src/Altinn.App.Core/Models/Layout/ModelBinding.cs

```diff
@@ -0,0 +1,26 @@
+namespace Altinn.App.Core.Models.Layout;
+
+/// <summary>
+/// Wrapper type for a model binding with optional data type specification
+/// </summary>
+public readonly record struct ModelBinding
+{
+    /// <summary>
+    /// The field in the model the binding is for
+    /// </summary>
+    public required string Field { get; init; }
```

Comment:
I have called it `property` in app-frontend, and so it currently unable to deserialize, so I guess we need to agree on what this should be called as well 😅

Reply:
ValidationIssue uses `Field`, so we use that.
https://github.com/Altinn/app-lib-dotnet/blob/06f0e26a11dca56d0ec784bf5fb376bf0ceefdb5/src/Altinn.App.Core/Models/Validation/ValidationIssue.cs#L49